### DISCUSSION TITLE
Don't have access to create Data Sets

### DISCUSSION THREAD

**Discussion by nihalalfred** [TestOptimal/MBT#9] [Q&A] [answered]
Hi, I have access to the community version, however, I am only able to create model and do not see the option to create data sets. 
Can you please help. 
my user id is : nihal.alfred@ly.st

**Comment by testoptimal-support** [comment, marked as answer]
Hi Nihal,

This question was asked previously, here is the answer:
The "DATASET" tab was removed from version 7.0.3 as part of the major overhaul effort to re-focus TestOptimal/MBT and ProMBT. It's replaced by MScript plugin "$DATA" for simplicity.

Basically you upload the tab delimited file for your dataset to model's "data" folder and use MScript $DATA to access /open the file.

Support
TestOptimal
